index.js

```diff
@@ -339,16 +352,140 @@ class Raspi extends events.EventEmitter {
     }
   }
 
+  i2cConfig(delay) {
+    if (this[i2c] === undefined) {
```

Comment:
This should be reworked so that ```this[i2c]``` is instantiated in the constructor. Then, in these I2C methods, we can do a check for ```this[i2c].alive``` instead, and throw an error if it's not with a message like ```I2C pins not in I2C mode. Once the mode is changed on an I2C pin, it cannot be changed back```.